src/parsec_client/operations/service_api_coverage.md

```diff
@@ -7,36 +7,38 @@ table.
+| [PsaDestroyKey](psa_destroy_key.md)                              | ❎  | ✅         | ✅     | ✅     | ✅             | ✅                |
+| [PsaExportKey](psa_export_key.md)                                | ❎  | ✅         | ❌     | ❌     | ❌             | ❌                |
+| [PsaExportPublicKey](psa_export_public_key.md)                   | ❎  | ✅         | ✅     | ✅     | ✅             | ❌                |
+| [PsaHashCompute](psa_hash_compute.md)                            | ❎  | ✅         | ❌     | ❌     | ❌             | ✅                |
+| [PsaHashCompare](psa_hash_compare.md)                            | ❎  | ✅         | ❌     | ❌     | ❌             | ✅                |
+| [PsaMacCompute](psa_mac_compute.md)                              | ❎  | ❌         | ❌     | ❌     | ❌             | ❌                |
+| [PsaMacVerify](psa_mac_verify.md)                                | ❎  | ❌         | ❌     | ❌     | ❌             | ❌                |
+| [PsaCipherEncrypt](psa_cipher_encrypt.md)                        | ❎  | ❌         | ❌     | ❌     | ❌             | ❌                |
+| [PsaCipherDecrypt](psa_cipher_decrypt.md)                        | ❎  | ❌         | ❌     | ❌     | ❌             | ❌                |
+| [PsaAeadEncrypt](psa_aead_encrypt.md)                            | ❎  | ✅         | ❌     | ❌     | ❌             | ❌                |
+| [PsaAeadDecrypt](psa_aead_decrypt.md)                            | ❎  | ✅         | ❌     | ❌     | ❌             | ❌                |
+| [PsaSignMessage](psa_sign_message.md)                            | ❎  | ❌         | ❌     | ❌     | ❌             | ❌                |
+| [PsaVerifyMessage](psa_verify_message.md)                        | ❎  | ❌         | ❌     | ❌     | ❌             | ❌                |
+| [PsaSignHash](psa_sign_hash.md)                                  | ❎  | ✅         | ✅     | ✅     | ✅             | ❌                |
+| [PsaVerifyHash](psa_verify_hash.md)                              | ❎  | ✅         | ✅     | ✅     | ✅             | ❌                |
+| [PsaAsymmetricEncrypt](psa_asymmetric_encrypt.md)                | ❎  | ✅         | ✅     | ✅     | ❌             | ❌                |
+| [PsaAsymmetricDecrypt](psa_asymmetric_decrypt.md)                | ❎  | ✅         | ✅     | ✅     | ❌             | ❌                |
+| [PsaRawKeyAgreement](psa_raw_key_agreement.md)                   | ❎  | ✅         | ❌     | ❌     | ❌             | ❌                |
+| [PsaGenerateRandom](psa_generate_random.md)                      | ❎  | ✅         | ❌     | ❌     | ❌             | ✅                |
+| [AttestKey](attest_key.md)                                       | ❌  | ❌         | ❌     | ❌     | ❌             | ❌                |
+| [PrepareKeyAttestationParams](prepare_key_attestation_params.md) | ❎  | ❎         | ❎     | ❌     | ❎             | ❎                |
```

Comment:
Should that be `PrepareKeyAttestation`? And now that it is generic, I guess it could be implemented on some other providers?